### DISCUSSION TITLE
How to use Tailwind CSS with pug-plugin

### DISCUSSION THREAD

**Discussion by a-gunderin** [webdiscus/pug-plugin#72] [General]
Hello @webdiscus !

I was trying to use Tailwind (https://github.com/tailwindlabs/tailwindcss) with pug-plugin but unfortunately I couldn't get the correct CSS to be generated.
Have you ever try to use Tailwind with pug-plugin?
My repo with pug-plugin and Tailwind: https://github.com/a-gunderin/webpack-starter

Regards.

**Comment by webdiscus** [comment]
The solution see please in the solved [issue](https://github.com/webdiscus/pug-plugin/issues/73#issuecomment-1479644111).
